docs: add note about command injection for helm example (#324)
argoproj-labs/argocd-vault-plugin: docs/usage.md

          value: -f values-dev.yaml -f values-dev-tag.yaml
```

> **Note: Bypassing the parameters like this can be dangerous in a multi-tenant environment as it could allow for malicous injection of arbitrary commands. So be cautious when doing something like in a production environment. Making sure that proper permissions and protections are very important when doing something liek this.** 

Reviewer: ```suggestion
**Note: Bypassing the parameters like this can be dangerous in a multi-tenant environment as it could allow for malicious injection of arbitrary commands. So be cautious when doing something like in a production environment. Ensuring proper permissions and protections is very important when doing something like this.** 
```

> **Note: Bypassing the parameters like this can be dangerous in a multi-tenant environment as it could allow for malicous injection of arbitrary commands. So be cautious when doing something like in a production environment. Making sure that proper permissions and protections are very important when doing something liek this.** 

Reviewer 2: ```suggestion
**Note: Bypassing the parameters like this can be dangerous in a multi-tenant environment as it could allow for malicious injection of arbitrary commands. So be cautious when doing something like in a production environment. Making sure that proper permissions and protections are enforced is very important when doing something like this.** 
```
